patches/minecraft/net/minecraft/server/level/ServerPlayerGameMode.java.patch

```diff
@@ -1,19 +1,17 @@
 --- a/net/minecraft/server/level/ServerPlayerGameMode.java
 +++ b/net/minecraft/server/level/ServerPlayerGameMode.java
-@@ -125,7 +_,15 @@
+@@ -125,7 +_,13 @@
        double d1 = this.f_9245_.m_20186_() - ((double)p_9282_.m_123342_() + 0.5D) + 1.5D;
```

Comment:
Ah, disregard this comment, I believe this is the eye height of the player, which is accounted for by the use of the eye position in `IForgePlayer#canInteractWith(BlockPos, double)` rather than the raw position of the player.